docs/user_guide/workloads/run_spark.md

```diff
@@ -131,8 +131,8 @@ Assigning the applicationId and the queue path are possible.
 ```
 --conf spark.kubernetes.executor.label.applicationId=application-spark-0001
 --conf spark.kubernetes.driver.label.applicationId=application-spark-0001
---conf spark.kubernetes.executor.label.queue=default.root.sandbox
---conf spark.kubernetes.driver.label.queue=default.root.sandbox
+--conf spark.kubernetes.executor.label.queue=root.default.sandbox
```

Comment:
Modification:
This queue path could be root.default.sandbox -> root.sandbox

Idea:
This same issue also appears in "i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md".
Could you modify that file?

Reply:
Thank you, @0yukali0, for review, also changed the corresponding zh-cn doc now.